ludwig/modules/loss_modules.py

```diff
@@ -509,3 +509,15 @@ def rmspe_loss(targets, predictions):
         )
     )
     return loss
+
+
+class SigmoidCrossEntropyLoss(nn.Module):
+    def __init__(self, **kwargs):
+        super().__init__()
+        self.loss_fn = nn.BCEWithLogitsLoss()
```

Comment:
does ths work on vectors of outputs like in the set case?

Reply:
I tested this out on a set output feature and it worked

Reply:
And what was the aggregation function? Were all the independent losses for each dimension of the vector summed or averaged? I believe they should be summed, as it would be equivalent to have N binary output features.

Reply:
Discussed offline